certify-sunbird-insurance.properties

```diff
@@ -1,41 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
 
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential,VehicleInsuranceCredential
 
-## -------------------------------------- Authentication & Authorization -----------------------------------------------
+  ##---------------------------------Life Insurance Credential Configurations------------------------------------------------------
 
-mosip.certify.security.auth.post-urls={}
-mosip.certify.security.auth.put-urls={}
-mosip.certify.security.auth.get-urls={}
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:9ae73412-5783-4194-acec-6d004006660c
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```

Comment:
Can we have this and other IPs changed to internal service urls ?

Reply:
can we change this after the release, because we haven't tested with internal urls

Reply:
ok